Car.cpp

```diff
@@ -14,4 +18,9 @@ void Car::tryToEvade()
 {
     setSpeed(0);
     std::cout << name << ": hello officer, what seems to be the problem?" << std::endl;
-}
\ No newline at end of file
+}
+
+void Car::setSpeed(int s)
+{
+    Vehicle::setSpeed(s);
+}
```

Comment:
You have not added any extra functionality to the base class implementation.  remove this function.
- [ ] revise

Reply:
done.